### DISCUSSION TITLE
add ElasticSIPTrunkingCapableIcon

### DISCUSSION THREAD

**Discussion by vankhoawin** [twilio-labs/paste#1118] [Request: Icon] [answered]
Hi team! TL;DR: the ask is for a `ElasticSIPTrunkingCapableIcon`.

There is a `ProductElasticSIPTrunkingIcon` but if I remember correctly, DSYS wants a distinction between a number having a product capability, and the product itself.

`ProductElasticSIPTrunkingIcon`:

![image](https://user-images.githubusercontent.com/6936736/106074080-1ce9be00-60c0-11eb-92ac-68254c84dabe.png)

Capability icons ([which are used in the IncomingPhoneNumber table](https://www.stage.twilio.com/console/phone-numbers/incoming)):

![image](https://user-images.githubusercontent.com/6936736/106074066-165b4680-60c0-11eb-8960-fc0e8df47990.png)

**Comment by serifluous** [comment, marked as answer]
Thanks for remembering, @vnguyen94 ! If you can, we'd definitely appreciate if you [contributed the icon to our repo](https://paste.twilio.design/icons/how-to-add-an-icon/#adding-the-icon-to-the-paste-repository).

For now, let's keep the icon art the same between the two SIP icons—you can duplicate the [ProductElasticSIPTrunkingIcon SVG](https://github.com/twilio-labs/paste/blob/main/packages/paste-icons/svg/ProductElasticSIPTrunking.svg), rename it to `ElasticSIPTrunkingCapable`, and then run the self-service steps to get the new icon.

If you'd prefer we do it, let us know, and we'll add it to an upcoming sprint.

**Reply by vankhoawin** [reply]
yup can do! https://github.com/twilio-labs/paste/pull/1119
